pkg/runtime/kubernetes/kubeadm.go

```diff
@@ -35,10 +35,20 @@ echo ${driver}`
 
 // k.getKubeVersion can't be empty
 func (k *KubeadmRuntime) setKubeadmAPIVersion() {
+	var kv KubeVersion
+	kv = kv.Version(k.getKubeVersion())
+	kv1150Flag, err := kv.Compare(kv.Version(V1150))
```

Comment:
We could add a prefix like "greaterThan" to make the context more readable, "kv1150Flag" => "greaterThanKV1150". But your way is ok.

Reply:
> We could add a prefix like "greaterThan" to make the context more readable, "kv1150Flag" => "greaterThanKV1150". But your way is ok.

I consider that I will use your way, I am also confusing when named this variable.

Reply:
In my perspective, It doesn't matter how you name it. The key is to make the context more readable.